packages/lesswrong/lib/routes.ts

```diff
@@ -502,8 +502,7 @@ if (taggingNameIsSet.get()) {
       name: 'taggingSubforumCustomName',
       path: `/topics/:slug/subforum`,
       componentName: 'TagSubforumPage',
-      hideFooter: true,
-      noPadding: true,
```

Comment:
These were both only used for this path so I have rolled them both into `fullscreen`